pkg/initiate/initiate.go

```diff
@@ -63,10 +60,13 @@ func init() {
 		}
 		_ = logger.Info(100, "Attempting to start exporter service")
 		go func() {
-			err = svc.Run(serviceName, &windowsExporterService{stopCh: StopCh})
+			err = svc.Run(serviceName, &windowsExporterService{})
 			if err != nil {
 				_ = logger.Error(102, fmt.Sprintf("Failed to start service: %v", err))
 			}
+			defer func() {
+				StopCh <- true
+			}()
```

Comment:
Good point, I made mistake. I was thinking I need to make sure that the signal is sent otherwise it could be that the exporter still runs while the communication to windows is broken.
But there is no way it does not reach the sending part except svc.Run would panic somehow. But then defer does not help anyways since the the exporter would die anyway.